docs/scos/dev/technology-partner-guides/202204.0/payment-partners/braintree/installing-and-configuring-braintree.md

```diff
@@ -195,11 +195,21 @@ All commands and conditions are located in `SprykerEco\Zed\Braintree\Communicati
 Default implementation for the checkout payment plugins should be added to `Pyz/Zed/Checkout/CheckoutDependencyProvider.php`.
 
 ```php
+protected function getCheckoutPreConditions(Container $container)
+{
+    $plugins = [
```

Comment:
if this variable has no value, the please just update doc to return array

Reply:
Fixed